modules/ROOT/pages/fd-connection-create.adoc

```diff
@@ -0,0 +1,59 @@
+= Add a Connection to the Configuration of a Connector
```

Comment:
I suggest titling the container topic "Adding" and the actual task topic "Add." 
Let's avoid "To <do this>" titles....